README.md

```diff
@@ -31,84 +31,84 @@ userPassword: readonly
 
-Either stop slapd and change the kidmd configuration to listen where slapd used to listen or change the clients to connect to where kidmd listens to migrate.
+Either stop slapd and change the idmd configuration to listen where slapd used to listen or change the clients to connect to where idmd listens to migrate.
 
-Assuming its all on the same server, this goes like this (you might want to remove existing ldif configuration with `rm /etc/kopano/kidm/ldif/main.d/*.ldif` before, to avoid conflicts).
+Assuming its all on the same server, this goes like this (you might want to remove existing ldif configuration with `rm /etc/libregraph/idm/ldif/main.d/*.ldif` before, to avoid conflicts).
 
 ```bash
-slapcat > /etc/kopano/kidm/ldif/main.d/10-main.ldif
+slapcat > /etc/libregraph/idm/ldif/main.d/10-main.ldif
 systemctl stop slapd
 systemctl disable slapd
-sed -i 's/^#ldap_listen = .*/ldap_listen = 127.0.0.1:389/' /etc/kopano/kidmd.cfg
-systemctl restart kopano-kidmd
+sed -i 's/^#ldap_listen = .*/ldap_listen = 127.0.0.1:389/' /etc/libregraph/idmd.cfg
+systemctl restart libregraph-idmd
 ```
 
 ### Extra goodies
 
 #### Template support
 
-All ldif files loaded by kdidmd support template syntax as defined in https://golang.org/pkg/text/template to allow auto generation and replacement of various values. You can find example templates in the `/usr/share/doc/kopano-kidmd/examples/` directory as well. All the `gen` commands output template syntax if applicable.
+All ldif files loaded by kdidmd support template syntax as defined in https://golang.org/pkg/text/template to allow auto generation and replacement of various values. You can find example templates in the `/usr/share/doc/libregraph-idmd/examples/` directory as well. All the `gen` commands output template syntax if applicable.
 
 #### Generate secure password hash using the `gen passwd` command
 
-Kidmd supports secure password hashing using ARGON2. To create such password hashes either use `gen newusers` or the interactive `gen passwd` which is very similar to `slappasswd` from OpenLDAP.
+idmd supports secure password hashing using ARGON2. To create such password hashes either use `gen newusers` or the interactive `gen passwd` which is very similar to `slappasswd` from OpenLDAP.
 
 ```bash
-kopano-kidmd gen passwd
+libregraph-idmd gen passwd
 New password:
 Re-enter new password:
 {ARGON2}$argon2id$v=19$m=65536,t=1,p=2$MaB5gX2BI484dATbGFyEIg$h2X8rbPowzZ/Exsz4W20Z/Zk54C30YnY+YbivSIRpcI
 ```
 
-#### Test kidmd
+#### Test idmd
 
-Since kidmd provides a standard LDAP interface, also standard LDAP tools can be used to interact with it for testing. Run `apt install ldap-utils` to install LDAP commandline tools.
+Since idmd provides a standard LDAP interface, also standard LDAP tools can be used to interact with it for testing. Run `apt install ldap-utils` to install LDAP commandline tools.
```

Comment:
This example is for Debian - i suggest to avoid distro specific instructions here and instead generalize it or if in doubt remove this section too.